src/content/web3/index.md

```diff
@@ -125,7 +125,7 @@ The Web3 ecosystem is young and quickly evolving. As a result, it currently depe
 
 Web3 is a young and evolving ecosystem. Gavin Wood coined the term in 2014, but many of these ideas have only recently become a reality. In the last year alone, there has been a considerable surge in the interest in cryptocurrency, improvements to layer 2 scaling solutions, massive experiments with new forms of governance, and revolutions in digital identity.
 
-We are only at the beginning of creating a better internet with Web3, but as we continue to improve the infrastructure that will support it, the future of the internet looks bright.
+We are only at the beginning of creating a better Web with Web3, but as we continue to improve the infrastructure that will support it, the future of the Web looks bright.
```

Comment:
```suggestion
We are only at the beginning of creating a better internet with Web3, but as we continue to improve the infrastructure that will support it, the future of the internet looks bright.
```

Reply:
Confusing -- the same sentence uses the two terms interchangeably "internet" and "Web3".